source/includes/rankings.md

```diff
@@ -405,8 +405,8 @@ Parameter         | Required | Value
 Field               | Description
 ------------------- | -----------
 **country**         | The country this ranking list is for; 2 letter country code.
-**genre_id**        | Genre ID (string). We force “overall” ID value for top level ranking lists.
-**rank_id**         | Ranking list type which serves also as identifier. Set by stores.
+**genre_id**        | String representing rankin list genre/category, as returned by the store. We force “overall” ID value for top level ranking lists.
```

Comment:
typo: String representing ranking genre/category, as defined by the store.
typo2: I think you're not using regular double quotes: "" for overall.

Reply:
Fixed all typos in https://github.com/AppMonsta/slate/commit/615c1afe0924c8b14ebb98b2843819b00445175f